packages/data-point-codemods/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "data-point-codemods",
+  "version": "1.7.0",
```

Comment:
Shouldn't we start this at `v0.1.0` or `v1.0.0`?

Reply:
because we use lerna and the way we deploy, we deploy the same version for each package on every deployment 

https://github.com/lerna/lerna#fixedlocked-mode-default

Reply:
Ew, but OK.